stories/base.css

```diff
@@ -0,0 +1,26 @@
+@font-face {
```

Comment:
replaced base.scss with just the fonts/icons we need loaded. Ideally, this would be inside resources/ folder that we could then enable/disable on the css resources side, but my webpack skills were failing at getting that working 😆

Reply:
Mine also are not great :D, I changed one thing though, I replaced usage of `../../dist/button.css` to use sass files again, because when I run storybook and do some changes on sass file, storybook throws error because it could not find css file (it rebuid the file on the fly and sotrybook doesn't like it at all)